apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/ReadModifyWriteRulesTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.V2.Tests
+{
+    public class ReadModifyWriteRulesTest
+    {
+        public static IEnumerable<object[]> InvalidFamilyNames { get; } = new List<object[]>
```

Comment:
I've found it easier to do this by creating a `List<object>` and then calling `.Select(t => new[] { t }).ToList()` at the end. I may put something in the ClientTesting library for this.

Alternatively, we can use `TheoryData<T>` which I haven't tried yet, but which looks like it should be simple.

Reply:
Thanks. Didn't know about `TheoryData<T>`. That cleans things up a bit. Also used it in the client tests to hold all invalid row keys.